examples/12_css/package.json

```
@@ -9,9 +9,9 @@
     "start": "waku start --with-ssr"
   },
   "dependencies": {
-    "@stylexjs/stylex": "0.4.1",
-    "@vanilla-extract/css": "1.14.0",
-    "classnames": "2.3.2",
+    "@stylexjs/stylex": "0.5.1",
+    "@vanilla-extract/css": "1.14.1",
+    "classnames": "2.5.1",
```

@sandren suggested to make it work with this specific version.
```suggestion
    "classnames": "2.3.2",
```